tests/integration/courses.test.js

```diff
@@ -2120,6 +2154,19 @@ describe('COURSES ROUTES - PUT /courses/{_id}', () => {
       expect(response.statusCode).toBe(403);
     });
 
+    it('should not update intra_holding course as user is coach but try to update company representative', async () => {
```

Comment:
plutot que not update on met plutot "return 403"

Reply:
done